### PR TITLE
Update types_strings.txt

### PR DESCRIPTION
**Pull request by strebski**
Fixed Polish strings, added some a new ones

### CODE REVIEW

**Comment by strebski** [comment]
@matkoniecz , @RicoElectrico Dear Colleagues, could you review my small improvement? :-) Thanks in advance!

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -11629,6 +11629,7 @@
     fi = Jalankulkualue
     it = Area pedonale
     mr = पादचारी झोन
+    pl = Obszar deptaka
```
Comment (matkoniecz) [inline data/strings/types_strings.txt:11632]: It is not ideal but I have no better wording here :(

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -12088,6 +12089,7 @@
     et = Sissesõidutee
     fi = Ajotie
     mr = चालकमार्ग
+    pl = Droga podjazdowa
```
Comment (matkoniecz) [inline data/strings/types_strings.txt:12092]:
```suggestion
    pl = Podjazd
```

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -16533,7 +16535,7 @@
     mr = पाण्याचा नळ
     nb = Vannkran
     nl = Waterkraan
-    pl = Woda pitna
+    pl = Kran z wodą
```
Comment (matkoniecz) [inline data/strings/types_strings.txt:16538]: What is selected for `man_made=water_tap amenity=drinking_water` ?

Comment (strebski) [reply data/strings/types_strings.txt:16538]: In current version we have two "Woda pitna" (for amenity.drinking_water & man_made.water_tap), so I changed the second one for "Kran z wodą" (water from it have not be for drinking)

Comment (matkoniecz) [reply data/strings/types_strings.txt:16538]: My point is that `man_made=water_tap amenity=drinking_water` should show up as "Woda pitna" (though that may require tweaks beyond translations)

Comment (strebski) [reply data/strings/types_strings.txt:16538]: Yes, understood, but you are thinking on tag combination. Types_strings correspond to adding a new POI via OM app, and doing this we can just use one, single tag (without combination). So we have to be clear with distinction for drinking water (-> amenity=drinking_water) and "just water" (-> man_made=water_tap), when we don't know nothing about water quality.

File: data/strings/types_strings.txt (hunk 2 above)
Comment (matkoniecz) [reply data/strings/types_strings.txt:12092]: Note that you can apply this change if considered as improvement

Comment (strebski) [reply data/strings/types_strings.txt:12092]: Your suggestion is OK for me ;-)

**Review by vng — APPROVED** [review]

**Pull request merged**